guacamole-common/src/main/java/org/apache/guacamole/GuacamoleAuthenticationRejectedException.java

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
This is not a javadoc comment and should not start with two asterisks. Checking the other license headers in the project, I found none that were written with two leading asterisks.